### PR TITLE
Updates to 2020 SOM page [#10685]

### PR DESCRIPTION
**Pull request by craigcook**
## Description
Adding four new cards, one small copy update, and various other little tweaks. It looks like more changes than it is because ended up messing with a lot of whitespace.

The key areas to look at are the four new cards and their corresponding modals.

## Issue / Bugzilla link
#10685 

## Testing
http://localhost:8000/foundation/annualreport/2020/

### CODE REVIEW

File: bedrock/foundation/templates/foundation/annualreport/2020/index.html [hunk 1]
```diff
@@ -233,74 +262,103 @@ <h2><span class="highlight green">Building a more joyful internet</span></h2>
...
+            class='has-modal',
+            attributes='data-modal-id="picture-in-picture"',
+            aspect_ratio='mzp-has-aspect-16-9',
+            link_url='#picture-in-picture',
+            image_url='img/foundation/annualreport/2020/blog-header-pip.jpg',
+          )}}
+        </div>
+
+        <div class="mzp-l-card-half">
+          {{ card(
```
Comment (craigcook) [inline bedrock/foundation/templates/foundation/annualreport/2020/index.html:305]: New card for Pocket Best Of 2021

File: bedrock/foundation/templates/foundation/annualreport/2020/index.html [hunk 2]
```diff
@@ -233,74 +262,103 @@ <h2><span class="highlight green">Building a more joyful internet</span></h2>
...
+            tag_label='Quality Content',
+            class='has-modal',
+            attributes='data-modal-id="pocket-best-of-2021"',
+            aspect_ratio='mzp-has-aspect-16-9',
+            link_url='#pocket-best-of-2021',
+            image_url='img/foundation/annualreport/2020/pocket-best-of-2021.jpg',
+          )}}
+
+          {{ card(
+            title='Falling in love with color',
```
Comment (craigcook) [inline bedrock/foundation/templates/foundation/annualreport/2020/index.html:317]: New card for Firefox colorways

File: bedrock/foundation/templates/foundation/annualreport/2020/index.html [hunk 3]
```diff
@@ -310,91 +368,123 @@ <h2>We put joy on your reading list</h2>
...
             aspect_ratio='mzp-has-aspect-16-9',
-            link_url='privacy-control',
+            link_url='#privacy-control',
             image_url='img/foundation/annualreport/2020/firefox-asset.png',
           )}}
         </div>
 
+        <div class="mzp-l-card-half">
+          {{ card(
+            title='Putting cookies where they belong',
```
Comment (craigcook) [inline bedrock/foundation/templates/foundation/annualreport/2020/index.html:466]: New card for Total Cookie Protection

File: bedrock/foundation/templates/foundation/annualreport/2020/index.html [hunk 4]
```diff
@@ -310,91 +368,123 @@ <h2>We put joy on your reading list</h2>
...
+            tag_label='Privacy',
+            class='has-modal',
+            attributes='data-modal-id="total-cookie-protection"',
+            aspect_ratio='mzp-has-aspect-16-9',
+            link_url='#total-cookie-protection',
+            image_url='img/foundation/annualreport/2020/total-cookie-protection.jpg',
+          )}}
+
+          {{ card(
+            title='Keeping your inbox at 0',
```
Comment (craigcook) [inline bedrock/foundation/templates/foundation/annualreport/2020/index.html:477]: New card for Firefox Relay premium

**Review by nathan-barrett — APPROVED** [review]
R+ Looks good to me - i'll merge once tests pass.

**Pull request merged**